core/framework/embedders_compat.py

```
@@ -0,0 +1,32 @@
+"""
+Backward Compatibility for Embedders
+
+This file provides backward compatibility for code that imports from
+core.framework.embedders. It redirects to the new core.embedders module.
+
+DEPRECATED: This will be removed after the restructure is complete.
+"""
+
+import warnings
+from core.embedders import *
```

Ruff (0.12.2)

`from core.embedders import *` used; unable to detect undefined names

(F403)